### DISCUSSION TITLE
Final project FIR 問題

### DISCUSSION THREAD

**Discussion by Ponponri** [bol-edu/HLS-SOC-Discussions#184] [Q&A] [answered]
請問期末改進 FIR 是基於 Lab4-2 的還是 Lab6 的， data length 一個是 600 一個是 11，謝謝。

**Comment by Russ420** [comment]
PYNQ 醒了沒 ??

**Comment by boledulab** [comment, marked as answer]
PYNQ is known for its reliability problem.  You are encouraged to use KV260. There are 10 more KV260, total about 20 KV260.

**Comment by boledulab** [comment]
@Ponponri  To balance with other workload, FIR workload is dataset at 64 data.
